### PR TITLE
Support ext option

### PR DESCRIPTION
**Pull request by nodaguti**
Hi!

I implemented `ext` option for `valid` and `invalid` test cases, which is filed in #5, and improved the ReadMe document.

### CODE REVIEW

File: test/fixtures/rule/fixer-rule-add.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
 // LICENSE : MIT
 "use strict";
+
+var RuleHelper = require("textlint-rule-helper").RuleHelper;
```
Comment (azu) [inline test/fixtures/rule/fixer-rule-add.js]: Prefer `const`  tha `var`

**Comment by azu** [comment]
@nodaguti Thanks! nice work :+1:

**Comment by nodaguti** [comment]
Thanks! :+1:

**Comment by azu** [comment]
Release [1.2.0](https://github.com/textlint/textlint-tester/releases/tag/1.2.0)

**Pull request merged**